docs_src/examples/Ch-ExamplesAddingModbusDevice.md

```diff
@@ -477,11 +480,12 @@ $ curl http://localhost:59882/api/v2/device/name/Modbus-TCP-Temperature-Sensor/A
 
 ## AutoEvent
 The AutoEvent is defined in the [[DeviceList.AutoEvents]] section of the device configuration file:
```

Comment:
```suggestion
The AutoEvent is defined in the autoEvents section of the deviceList in configuration file:
```

Reply:
Good catch. But is is the `device definition file`